### DISCUSSION TITLE
Do you have any requests if I were to fork this repo?

### DISCUSSION THREAD

**Discussion by ryanzec** [lukasoppermann/design-tokens#297] [Q&A]
@lukasoppermann This next block is to convey my reason for wanting to continue this work (to an extent), feel free to skip it to se my quest at the bottom.

So while I agree that the source of truth for design tokens should be a file not tied to a specific application, I disagree that manually modification of that file is the best way to maintain it. If I want to change a color, I want to be able to see the effect that would have across by entire time which I find much easier to do in my design tool of choice. I think it is easier to tweak the value in Figma and review how that effect things rather than modify a `json` file -> import to figma -> check change which might result in a longer time because there are more steps is tweaking the value to seeing changes. Of course everyone is different so I don't expect this flow to be optimal for everyone.

With that said, I would like to fork and maintain a version of this plugin (and older version as the current one has broken the work I did for exporting modes). I will obviously reference this repo as the starting point for it (unless you don't want me to) however do you have requests for me in doing this? 

Like
- don't create a public repo of it
- don't publish to figma community
- etc.

**Comment by lukasoppermann** [comment]
Hey,

yeah, I also design my colors in Figma, I just copy it over.

I would appreciate it if you would NOT publish it to the community.
But feel free to keep the project public so people can download it and just run it as a local version.

What are the big things you want to change?

**Comment by ryanzec** [comment]
It is not that I want to change anything, it is the fact the work I did for exporting modes does not work anymore. While I would have looked into fixing it, if this project is not longer being actively support, I would rather just keep the version that works for me and fix issues as I come up with them.

**Reply by lukasoppermann** [reply]
Fair enough. If you manage to fix it in a way that could be contributed as a PR as well, I would appreciate this.

I am still merging PRs and maintaining it as much as I can, so any fix is always welcome.

**Comment by hayawata3626** [comment]
@lukasoppermann 

Thank you for using this wonderful tool. I have some questions I'd like to ask🙏

From the following statement, I understand that there will be no active development of new features in the future. Are there any plans to recruit maintainers?


> This means: I don't have capacity to add any features or really improve stuff. Bugfixes may be added if time allows. I am also happy to discuss and merge PRs.


Also, I'm not quite clear on the following point: Is it not allowed to change the name of the plugin after forking and then publish it to the Figma community? (Since this plugin is managed under the MIT License, I'm a bit concerned about this aspect.) If it's not allowed, would it be okay to keep the plugin limited to internal organization use only?


> I would appreciate it if you would NOT publish it to the community. But feel free to keep the project public so people can download it and just run it as a local version.


I would like to express my gratitude again for always being able to use this plugin.

**Reply by lukasoppermann** [reply]
> From the following statement, I understand that there will be no active development of new features in the future. Are there any plans to recruit maintainers?

No, I have no plans. But if you or anyone is interested, I am open to this. I'd love to get some folks into this so it can get more updates. I just personally don't have the capacity to put enough work into it to develop new features.

> Also, I'm not quite clear on the following point: Is it not allowed to change the name of the plugin after forking and then publish it to the Figma community? (Since this plugin is managed under the MIT License, I'm a bit concerned about this aspect.) 
Since it is MIT I guess it is allowed. But since this project is still alive and I still contribute fixes, it feels bad if someone just takes it and republishes it under a different name with no or very minor changes on Figma. That being said, legally this would probably be fine. 

> If it's not allowed, would it be okay to keep the plugin limited to internal organization use only?
Yes, for sure.

Just to clarify this, I am still fixing issues were possible and I accept PRs and merge them. And if there is someone who wants to join the maintainer team, I am happy for that as well.

**Comment by hayawata3626** [comment]
@lukasoppermann 
> No, I have no plans. But if you or anyone is interested, I am open to this. I'd love to get some folks into this so it can get more updates. I just personally don't have the capacity to put enough work into it to develop new features.

I am interested in that! (I'm not sure if I can make time for it regularly, though.)

> Just to clarify this, I am still fixing issues were possible and I accept PRs and merge them. And if there is someone who wants to join the maintainer team, I am happy for that as well.

If I can resolve all the issues you are currently working on, are you allowed to fork it and publish it as my own plugin?
